ABaseEntity/ABaseEntity.cpp

```diff
@@ -12,7 +12,8 @@ ABaseEntity::ABaseEntity() : IBaseEntity() {
 
 void ABaseEntity::BeginPlay() {
     g_pGlobals->checkReset();
-    AActor::BeginPlay();

```

Comment:
It's not good practice to explicitly say the name of the class. It's a lot better to say `Super`